server/features/etcd_features.go

```diff
@@ -95,6 +103,7 @@ var (
 		"experimental-txn-mode-write-with-shared-buffer": TxnModeWriteWithSharedBuffer,
 		"experimental-enable-lease-checkpoint":           LeaseCheckpoint,
 		"experimental-enable-lease-checkpoint-persist":   LeaseCheckpointPersist,
+		"experimental-enable-set-member-localaddr":       SetMemberLocalAddr,
```

Comment:
It's added in main only previously, we don't need to backward compatible with anything, so let's just remove it
```suggestion
```

Reply:
Got it, thanks!